src/backend/src/routes/reimbursement-requests.routes.ts

```diff
@@ -24,6 +24,8 @@ reimbursementRequestsRouter.get('/reimbursements/current-user', ReimbursementReq
 
 reimbursementRequestsRouter.get('/reimbursements', ReimbursementRequestController.getAllReimbursements);
 
+reimbursementRequestsRouter.post('/edit-vendors', nonEmptyString(body('name')), ReimbursementRequestController.editVendor);
```

Comment:
We want to add the `validateInputs` function here when there's at least one input from this endpoint payload. The `validateInputs` function makes sure there is no error from the endpoint request.